app/classes/report/mycoportal.rb

```diff
@@ -1,174 +1,341 @@
 # frozen_string_literal: true
 
+require "haversine"
```

Comment:
Added to fix `uninitialized constant Report::Mycoportal::Haversine `when manually testing.
That's puzzling because tests work fine without this line.